spec/Dockerfile.puppetdb

```diff
@@ -1,13 +1,9 @@
-FROM puppet/puppetdb:6.5.0
+FROM puppet/puppetdb
 
 ARG hostname="bolt-puppetdb"
```

Comment:
This can be supplied in `docker-compose.yml`

Reply:
Oh, added to docker-compose.yaml. need to delete i here.